### PR TITLE
Add distance and usage backend routes

### PR DESCRIPTION
**Pull request by tedim52**
-Added routes to compute distance, and usage using a mock company object in app.py
-Added calculate function in tree-script.js to fetch backend data, currently logs it to console but response needs to be validated then displayed on frontend somehow

### CODE REVIEW

**Comment by tedim52** [comment]
This PR has the frontend integrated with the backend for Company Hierarchy Tree feature. Big PR but mainly merging files, loading in altered data formats, and fixing imports.
Screen Shot 2020-07-29 at 1 28 17 PM

File: core/resource_provisioning.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # This file contains some useful functions for metrics regarding provisioning
 import pickle
-from employee import Employee
-from resource import Resource
+from .employee import Employee
```
Comment (DeanAlvarez) [inline core/resource_provisioning.py:3]: rip relative imports :'(

File: core/utilities/user_resource_map.py [hunk 2]
```diff
@@ -1,39 +1,36 @@
 # Created by Kayla Patterson 07/16/2020
 # This file maps each user to their perspective
-# resources and exports it as a pickle file
```
Comment (DeanAlvarez) [inline core/utilities/user_resource_map.py:3]: Do you use pylint or something? It seems like there is some auto formating going on here

**Review by DeanAlvarez — APPROVED** [review]
approved

File: core/resource_provisioning.py (hunk 1 above)
Comment (tedim52) [reply core/resource_provisioning.py:3]: for real

File: core/utilities/user_resource_map.py (hunk 2 above)
Comment (tedim52) [reply core/utilities/user_resource_map.py:3]: I think I did that haha, I always put a space between the hashtag and comment i dont know why

**Pull request merged**